scripts/projectView.js

```diff
@@ -5,7 +5,6 @@ projectsView.handleMainNav = function() {
     e.preventDefault();
     $('.page-content').hide();
     var $clickEvent = $(this);
-    console.log($clickEvent);
     $('[id="' + $clickEvent.attr('data-content') + '"]').show();
```

Comment:
look into concatenating the # and the variable $clickEvent. to select with the id. for readability purposes.